### PR TITLE
Task 10.3

### PR DESCRIPTION
**Pull request by AdamPawlinski**
Task 10.3 finished, I hope so :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,12 +3,52 @@
 <head>
 <meta charset="utf-8">
 	<title>JQuery carousel</title>
+	<link href="https://meyerweb.com/eric/tools/css/reset/reset.css" rel="stylesheet" type="text/css"/>
+	<link href="css/style.css" rel="stylesheet" type="text/css"/>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script defer src="https://use.fontawesome.com/releases/v5.0.9/js/all.js" integrity="sha384-8iPTk2s/jMVj81dnzb/iFR2sdA7u06vHJyyLlAd4snFpCl/SnyUjRrbdJsw1pGIl" crossorigin="anonymous"></script>
```
Comment (matwjablonski) [inline index.html]: scripts move to the end of document

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  function moveSlide(){
+    var firstItem = carouselList.find("li:first");
+    var lastItem = carouselList.find("li:last");
+    lastItem.after(firstItem);
+    carouselList.css({marginLeft:0});
+  }
+
+  function carouselStart(){
+    interval;
```
Comment (matwjablonski) [inline js/script.js]: unnecessary?

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    clearInterval(interval);
+    carouselList.animate({marginLeft:-500}, 1000, moveSlide);
+    setInterval(changeSlide, 3000);
+  });
+
+  $('.slides-number > a').on('click', function(){
+    clearInterval(interval);
+  })
+
+  carouselStart();
```
Comment (matwjablonski) [inline js/script.js]: unnecessary

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    var firstItem = carouselList.find("li:first");
+    var lastItem = carouselList.find("li:last");
+    firstItem.before(lastItem);
+    carouselList.css({marginLeft:0});
+  }
+
+  $('#left').on('click', function(){
+    clearInterval(interval);
+    carouselList.animate({marginLeft:500}, 1000, moveSlidePrev);
+    setInterval(changeSlide, 3000);
```
Comment (matwjablonski) [inline js/script.js]: setInterval return an  id - when you clean your interval - it's ok, but for current moment

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/script.js (hunk 4 above)
Comment (AdamPawlinski) [reply js/script.js]: I cant't understand what you mean? What id? I commented out this line with setInterval.

File: js/script.js (hunk 3 above)
Comment (AdamPawlinski) [reply js/script.js]: I cut out carouselStart but I can't understand why it works without starting function?

**Review by AdamPawlinski — COMMENTED** [review]
changed but I've got questions.

File: js/script.js (hunk 4 above)
Comment (matwjablonski) [reply js/script.js]: every function return something, setInterval returns an id of interval. your new setInterval should be assign to previous variable

File: js/script.js (hunk 3 above)
Comment (matwjablonski) [reply js/script.js]: it is called in setInterval as callback

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
setInterval assigned and also setIntaerval and setTimeout added to click on slides-number

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**